### ISSUE TITLE
For some reason pyppasearch.py finds fewer packages than Y PPA Manager

### ISSUE THREAD

**Issue by N0rbert** [anenasa/pyppasearch#1]
First of all I would like to thank you for your application. It is useful, and may replace search on https://launchpad.net/ubuntu/+ppas which [does not have the filter](https://pad.lv/1820387) for searching for relevant Ubuntu version/suite.

For long time I'm using [Y PPA Manager](https://launchpad.net/~webupd8team/+archive/ubuntu/y-ppa-manager) to find PPAs locally.
And what is interesting it finds more packages than your script. If I understand [things correctly](https://bazaar.launchpad.net/~nilarimogard/y-ppa-manager/stable/view/head:/y-ppa-search#L402) both applications parse output from https://launchpad.net/ubuntu/+ppas URL. 
I'm running Ubuntu MATE 18.04.6 LTS with all updates and only Y PPA Manager PPA added to the system. Here I see the following for `i3-gaps` package search:

```
cd ~/Downloads
wget -c https://raw.githubusercontent.com/anenasa/pyppasearch/main/pyppasearch.py
sudo apt-get update
sudo apt-get install python3-bs
chmod +x pyppasearch.py

./pyppasearch.py -c Bionic i3-gaps
i3-gaps 4.17.1-0york0~18.04 jonathonf/i3 Bionic
Search is finished.
```

![y-ppa-manager_i3-gaps_bionic](https://user-images.githubusercontent.com/309456/144422603-b4458bab-f02c-46b5-96b2-2f3919f6b9d4.png)

Could you please elaborate why your script shows 1 line, while Y PPA Manager shows 12?

**Comment by anenasa** [comment]
Yes, this is a known problem. This is because my script only check for source name from package details, while Y PPA Manager [looks for package name](https://bazaar.launchpad.net/~nilarimogard/y-ppa-manager/stable/view/head:/y-ppa-search#L234). I do plan to fix this, but I do not have time.

**Comment by anenasa** [comment]
Closed by #2.
